audiorecorder

```diff
@@ -229,17 +216,17 @@ FILTER_CHAIN="asplit=6[out1][a][b][c][d][e],\
 [z]drawtext=fontfile=/Library/Fonts/Andale Mono.ttf: text='%{pts \\: hms}':x=460: y=50:fontcolor=white:fontsize=30:box=1:boxcolor=0x00000000@1[fps],[fps]fps=fps=30[out0]"
 
 if [ "${runtype}" = "passthrough" ] ; then
-    ffmpeg -f avfoundation -i "none:"${DEVICE_NUMBER}"" -f wav -c:a pcm_s16le -ar 44100 - |\
+    rec -r "${SAMPLE_RATE_NUMERIC}" -b 32 -L -e signed-integer --buffer 131072 -p | ffmpeg -i - -f wav -c:a pcm_s16le -ar 44100 - |\
     ffplay -window_title "Audio Recorder" -f lavfi \
     "amovie='pipe\:0',${FILTER_CHAIN}"
     stty sane
     exit
 fi
 
 _get_ITEM_ID
-
+rm -r PIPE2REC
```

Comment:
Is there always a file to remove? If not, I would suggest to trap the error message and redirect `>/dev/null`